### ISSUE TITLE
Python 3.6 is not compatible

### ISSUE THREAD

**Issue by indonoso** [facebookresearch/PyTorch-BigGraph#162]
## Steps to reproduce

  1. Install library on an environment with python 3.6
  2. Train with a config that uses GPU

## Observed Results

  * You will get `AttributeError: module 'time' has no attribute 'monotonic_ns'` from raised at line 81, file `train_gpu.py`
``` python
class TimeKeeper:
    def __init__(self):
        self.t = self._get_time()
        self.sub_ts = {}

    def _get_time(self) -> float:
        return time.monotonic_ns() / 1e9  # here
```

## Expected Results

  * To run the training process without any errors.
  * This happens because `time.monotonic_ns()` is not part of the [`time` module until python version 3.7](https://docs.python.org/3/library/time.html#time.monotonic_ns)

## Relevant Code

Not relevant to the issue

**Comment by lw** [comment]
Thanks for reporting this, good catch! We used to test PBG with Python 3.6 at the beginning but the GPU mode is recent and I suspect that we only ever ran it with Python 3.7. Do you think you could send a pull request to fix this? That would be great!

**Comment by lw** [comment]
Closed by #163.
